.github/workflows/examples-nrfconnect.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-nrf-platform:0.5.56
+            image: connectedhomeip/chip-build-nrf-platform:0.5.58
```

Comment:
Could we update globally instead?

Also in particular `integrations/cloudbuild/smoke-test.yaml` and `integrations/cloudbuild/build-all.yaml` also will try to build nrf and would fail if expected interface is not correct (and maybe we should update vscode images as well).

Reply:
Ok, done.